src/main/java/org/folio/inventory/domain/AsynchronousCollection.java

```diff
@@ -11,9 +11,15 @@
 import org.folio.inventory.exceptions.InternalServerErrorException;
 
 public interface AsynchronousCollection<T> {
-  void empty(Consumer<Success<Void>> completionCallback,
+  void empty(String cqlQuery,
```

Comment:
The name empty no longer makes sense given that the CQL query parameter allows for only part of the collection to be affected.